nextflow/restart.nf

```diff
@@ -0,0 +1,63 @@
+#!/usr/bin/env nextflow
+
```

Comment:
Ok, I will remove it. But this restart script based on the nextflow is better than a standalone script. 

1. is not relevant to the restart topic because this script only cares about workdir, which is the intermediate fine; 
2. We can not afford to rerun since each training takes more than one day. It also needs to find out which tasks failed and restart them separately(which here must use a script);
3. no matter what language I used here(Groovy / script), it still need to run by using nextflow. After restarting, it can link final result to publicDir. 
4. Since I don't find an easy way to group tasks and runs in a same workdir, it is better to use different workdir for different experiment...

Reply:
Ok, consider this resolved then. For the sake of restarting, I think NF logs the tasks for each run, see [`nextflow log [runname | sessionid]`](https://www.nextflow.io/docs/latest/cli.html#log), that's what I mean by "workdirs of that run".